src/layouts/navigation.ts

```diff
@@ -9,9 +9,9 @@ export const getNavigation = (section) => {
           href: "/dev/intro",
         },
         {
-          title: "Send Tokens",
+          title: "Cross-Chain Token Transfer",
```

Comment:
Did we agree on this naming/term? Have you had a chance to run it by Stephen as we discussed?

Reply:
@ffe9f8

Reply:
Yup, Stephen approved -- we should use "cross-chain" here to maximize searchability.

@StephenFluin